Author: Nicholas Bamber <nicholas@periapt.co.uk>
Debian-Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=540153
Subject: tests not getting started on kFreeBSD
 As per #670722 I found that the test socket created to test
 file name truncation was barfing even for the shortened form.
Forwarded: no
Last-Update: 2012-04-28
Index: mysql-5.5.29/mysql-test/lib/My/Platform.pm
===================================================================
--- mysql-5.5.29.orig/mysql-test/lib/My/Platform.pm	2013-01-02 15:38:08.271735374 -0800
+++ mysql-5.5.29/mysql-test/lib/My/Platform.pm	2013-01-02 15:38:13.707416629 -0800
@@ -113,6 +113,8 @@
   # Similarly the path length is hidden.
   # See Debian bug #651002
   return 0 if ($^O eq 'gnu');
+  # See Debian bug #670722 - failing on kFreeBSD even after setting short path
+  return 0 if length $path < 40;
 
   require IO::Socket::UNIX;
 
